tools/perf/brave_page_sets/brave_system_health/system_health_stories.py

```diff
@@ -47,6 +47,29 @@ def GenerateStoryDescription(cls):
     return 'Load %s' % cls.URL
 
 
+class _BraveMultiTabLoadingStory(_BraveLoadingStory):
+
+  TABS_COUNT = 10
+
+  def RunPageInteractions(self, action_runner):
+
+    def load_and_wait(action_runner):
+      action_runner.tab.WaitForDocumentReadyStateToBeComplete()
+      self._DidLoadDocument(action_runner)
```

Comment:
What `_DidLoadDocument()` actually inside?

Reply:
Sorry, I didn't quite get that! Do you mean its definition? It just calls the implementation from `_BraveLoadingStory`.

Reply:
The question was what it actually does (taking into account that we don't call the matching `_Measure()`

Reply:
I actually just want to have the scrolling functionality (if needed for sites other than `example.com`), which is why I called it here.